run-ci.sh

```diff
@@ -24,4 +21,14 @@ for recipe_name in $changed_recipes; do
     fi
 done
 
+# if the tooling in ./package-build changed test a couple 'interesting' recipes:
+changed_tooling=$(echo "$CHANGED_FILES" | (grep -Po '(?<=^package-build/)[a-z0-9].*' || true))
```

Comment:
Doesn't the `|| true` mean that it will do this regardless?

Reply:
I stole the pattern from the previous line for searching for changed recipes, but the `|| true` keeps grep from giving an error returncode when it doesn't find any matches.  And in that case (no matches), `changed_tooling` will be the empty string, so `[ -n "$changed_tooling"]` on the next line will be false.